editor/scene_tree_dock.cpp

```diff
@@ -2616,7 +2616,7 @@ void SceneTreeDock::attach_script_to_selected(bool p_extend) {
 	}
 
 	script_create_dialog->connect("script_created", this, "_script_created");
-	script_create_dialog->connect("popup_hide", this, "_script_creation_closed");
+	script_create_dialog->connect("popup_hide", this, "_script_creation_closed", varray(), CONNECT_DEFERRED);
```

Comment:
I guess it could be `CONNECT_ONESHOT`, and then the `disconnect` can be removed?

Reply:
Ping.

Reply:
Force pushed an amend that does that.